tools/setup.py

```diff
@@ -0,0 +1,55 @@
+#!/bin/python3
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import date
+from glob import glob
+from setuptools import setup
+import os
+
+version = '0.0.1.dev' + date.today().strftime("%Y%m%d")
+project_name = 'gematria'
+
+SO_DIRECTORIES = ['./pybind11_abseil', './gematria']
+
+SHARED_OBJECTS = []
```

Comment:
Nit: This is not a constant, so it should use the usual "variable" name `shared_objects`.